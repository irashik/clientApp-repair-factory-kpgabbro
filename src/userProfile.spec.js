import React from 'react';

import ProfileUserComponent  from './userProfile';

       


describe("<ProfileUserComponent/>", () => {
    test('should render correctly empty list', () => {

        const component = shallow(<ProfileUserComponent />)
        expect(component).toMatchSnapshot();
    });



    test.skip('should render correctly full list', () => {


        // добавь данные ...

        const component = shallow(<ReadModuleList />)
        expect(component).toMatchSnapshot();
    });




});



