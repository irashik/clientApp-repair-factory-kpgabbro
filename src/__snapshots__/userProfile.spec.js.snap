// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileUserComponent/> should render correctly empty list 1`] = `
<Container
  fluid={true}
>
  <Row
    className="justify-content-md-center"
    noGutters={false}
  >
    <h2>
      Профиль пользователя
    </h2>
  </Row>
  <Row
    noGutters={false}
  >
    <Form
      inline={false}
      onSubmit={[Function]}
    >
      <FormGroup
        controlId="formBasicName"
      >
        <FormLabel
          column={false}
          srOnly={false}
        >
          Ваши Имя и Фамилия
        </FormLabel>
        <FormControl
          onChange={[Function]}
          placeholder="Фамилия и имя"
          type="text"
          value=""
        />
      </FormGroup>
      <FormGroup
        controlId="UserPosition"
      >
        <FormLabel
          column={false}
          srOnly={false}
        >
          Ваша должность
        </FormLabel>
        <FormControl
          onChange={[Function]}
          placeholder="Должность"
          type="text"
          value=""
        />
      </FormGroup>
      <FormGroup>
        <br />
        <Button
          active={false}
          disabled={false}
          id="changeProfileBtn"
          type="submit"
          variant="primary"
        >
          Обновить данные
        </Button>
      </FormGroup>
    </Form>
    <br />
    <br />
    <br />
    <Row
      className="justify-content-md-center"
      noGutters={false}
    >
      <h4>
        Изменение пароля
      </h4>
    </Row>
    <Form
      inline={false}
      onSubmit={[Function]}
    >
      <FormGroup
        controlId="formBasicPassword"
      >
        <FormLabel
          column={false}
          srOnly={false}
        >
          Введите новый пароль
        </FormLabel>
        <FormControl
          onChange={[Function]}
          placeholder="New Password"
          type="password"
          value=""
        />
      </FormGroup>
      <FormGroup
        controlId="formBasicPassword2"
      >
        <FormControl
          onChange={[Function]}
          placeholder="Retry new password"
          type="password"
          value=""
        />
      </FormGroup>
      <br />
      <Button
        active={false}
        disabled={false}
        id="changePasswordBtn"
        type="submit"
        variant="secondary"
      >
        Сменить пароль
      </Button>
    </Form>
  </Row>
</Container>
`;
